mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -50,123 +54,311 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BasicDocSaver.class);
 
+    private int batchSize=64;
```

Comment:
Rational behind 64?  It's hard coded, is it worth exposing and making configurable via puppet?  If yes we could create an issue to fix do that later.